sway-core/src/ir_generation/const_eval.rs

```diff
@@ -1149,6 +1313,82 @@ fn const_eval_intrinsic(
                 span: intrinsic.span.clone(),
             })
         }
+        // Buffer starts ((ptr, 1024), 0)
```

Comment:
The comment is misleading, because the initial buffer size is zero because of `vec![]`. We should either fix the comment or the initial buffer size.

Reply:
Removed. It is not helping anything actually.